stack/stack.go

```diff
@@ -103,3 +103,16 @@ func fetchYAML(address *url.URL) ([]byte, error) {
 
 	return resBytes, err
 }
+
+func WriteYAMLData(services *Services, fileName string) error {
```

Comment:
Have we got any tests on the serialization on its own?

Reply:
No but I will add some.

Reply:
@ericstoekl can you rename it to WriteYAMLDataForServices, in #87 I made some changes, ParseYAMLData is for general parsing, and ParseYAMLDataForServices is used for services

Reply:
Thanks for the heads up @itscaro. If #87 gets merged before this one, I'll make that update.